xray/commands/audit/scan.go

```diff
@@ -32,7 +32,7 @@ type ScanCommand struct {
 	serverDetails *config.ServerDetails
 	spec          *spec.SpecFiles
 	threads       int
-	// The location on the local file system of the downloaded Xray's indexer.
+	// The location on the local file system of the downloaded Xray indexer binary.
```

Comment:
// The location on the local file system of the downloaded Xray indexer binary.
-->
// The location of the downloaded Xray indexer binary on the local file system.